shared_model/backend/protobuf/queries/proto_get_account_transactions.hpp

```diff
@@ -48,6 +49,10 @@ namespace shared_model {
         return account_transactions_->account_id();
       }
 
+      const Pager &pager() const override {
+        return *pager_;
```

Comment:
Not covered by tests

Reply:
I think integration test is needed for all commands with shared model.

Reply:
Do you have unit tests for it?

Reply:
Yes. These are tests for this field.
1. https://github.com/hyperledger/iroha/pull/841/files#diff-9b09a2c1edeb8a28892ec5e069b23a00R275
2. https://github.com/hyperledger/iroha/pull/841/files#diff-ca6b0d01ed448398ba40ff044dc7dab3R82

Reply:
Hmm... but, codecov shows red.